pubsub/subscriptions/main.go

```diff
@@ -1,307 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Need to pin the sample version in docs before we can delete this file.

Reply:
Done